Login.xaml.cs

```diff
@@ -0,0 +1,258 @@
+                            MainWindow.InsertingLogTextToLogFile("Password is incorrect. Please try again." + " (" + DateTime.Now.ToString() + ")");
+                            SentrySdk.CaptureMessage("Password is incorrect. Please try again.");
+                        }
+                    }
+                }
+                catch (Exception ex)
+                {
+                    SentrySdk.CaptureMessage(ex.Message);
+                }
+            }
+        }
+
+        private void CreatingLoginCredentialFile()
+        {
+            var username = "admin";
+            var password = "adminadmin";
+
+            //string pathDir = System.IO.Path.Combine(Directory.GetParent(System.IO.Directory.GetCurrentDirectory()).Parent.FullName + "\\Credential");
+            string pathDir = AppDomain.CurrentDomain.BaseDirectory + "\\Credential";
+            if (!Directory.Exists(pathDir))
+            {
+                Directory.CreateDirectory(pathDir);
+            }
+
+            //string path = Directory.GetParent(System.Reflection.Assembly.GetExecutingAssembly().Location).FullName;
+            //string path = System.IO.Path.Combine(Directory.GetParent(System.IO.Directory.GetCurrentDirectory()).Parent.FullName + "\\Credential", "LoginCredential.xml");
+            //string fileName = System.IO.Path.Combine(path, "LoginCredential.xml");
+            string path = AppDomain.CurrentDomain.BaseDirectory + "\\Credential\\LoginCredential.xml";
+
+            try
+            {
+                var Status_DefaultPasswordExist = CheckDefaultPasswordExist();
+
+                if (File.Exists(path))
+                {
+                    if (Status_DefaultPasswordExist == true)
+                    { }
+                }
+                else
+                {
+                    XmlWriterSettings xmlWriterSettings = new XmlWriterSettings();
+                    xmlWriterSettings.Indent = true;
+                    xmlWriterSettings.NewLineOnAttributes = true;
+                    using (XmlWriter xmlWriter = XmlWriter.Create(path, xmlWriterSettings))
+                    {
+                        xmlWriter.WriteStartDocument();
+                        /*LoginCredential Config*/
+                        xmlWriter.WriteStartElement("LoginCredential");
+                        xmlWriter.WriteElementString("UserName", Helper.Crypto.Encrypt(username));
+                        xmlWriter.WriteElementString("Password", Helper.Crypto.Encrypt(password));
+                        xmlWriter.WriteEndElement();
+
+                        xmlWriter.WriteEndDocument();
+                        xmlWriter.Flush();
+                        xmlWriter.Close();
+                    }
+                }
+            }
+            catch(Exception ex)
+            {                
+                SentrySdk.CaptureMessage(ex.Message);
+            } 
+        }
+
+        private bool CheckDefaultPasswordExist()
```

Comment:
A lot of this function looks like it's been exactly **copy-pasted** from other parts of this same file! Compare `CheckDefaultPasswordExist` and `GetPasswordFromXML`. That's a really bad sign...